src/modules/emotes/index.js

```diff
@@ -20,6 +21,7 @@ class EmotesModule {
       frankerfacezGlobalEmotes,
       frankerfacezChannelEmotes,
       seventvChannelEmotes,
+      seventvGlobalEmotes,
```

Comment:
order of this list matters so global emotes of each provider needs to come before channel emotes of each provider so channel emotes take preference